# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1206638804 14400
#      Thu Mar 27 13:26:44 2008 -0400
# Branch release-3-0-x
# Node ID a99c93f834045d68bff74b4c4102f2fd97bcb47a
# Parent  6e7e29c2155b7755807c9f89f0cb511ea8a55347
__axis_label__: use name of caller in error message

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,12 @@
+2008-03-27  John W. Eaton  <jwe@octave.org>
+
+	* plot/__axis_label__.m: Use name of caller in error message.
+
 2008-03-26  John W. Eaton  <jwe@octave.org>
 
 	* set/ismember.m: Set size of idx output correctly for empty args.
 	New tests.
 
 	* general/logical.m: Correctly handle empty args.  New tests.
 
 2008-03-24  Thomas Weber  <thomas.weber.mail@gmail.com>
diff --git a/scripts/plot/__axis_label__.m b/scripts/plot/__axis_label__.m
--- a/scripts/plot/__axis_label__.m
+++ b/scripts/plot/__axis_label__.m
@@ -33,12 +33,12 @@ function retval = __axis_label__ (caller
     endif
 
     h = __go_text__ (ca, "string", txt, "rotation", rot, varargin{:});
     set (ca, caller, h);
     if (nargout > 0)
       retval = h;
     endif
   else
-    error ("%s: expecting first argument to be character string");
+    error ("%s: expecting first argument to be character string", caller);
   endif
 
 endfunction
